pages/deployment/[uuid].tsx

```diff
@@ -196,20 +202,20 @@ export default function Deployment() {
               <div>
                 # Login to Docker (your token can be found on the <Link href='/settings'>settings</Link> page)
               </div>
-              <CodeLine line={`docker login ${uiConfig?.registry.host} -u ${currentUser!.id}`} />
+              <CodeLine index={1} line={`docker login ${uiConfig?.registry.host} -u ${currentUser?.id}`} />
               <br />
 
               <div># Pull model</div>
-              <CodeLine line={`docker pull ${deploymentTag}`} />
+              <CodeLine index={2} line={`docker pull ${deploymentTag}`} />
               <br />
 
               <div># Run Docker image</div>
-              <CodeLine line={`docker run -p 9999:9000 ${deploymentTag}`} />
+              <CodeLine index={3} line={`docker run -p 9999:9000 ${deploymentTag}`} />
               <div># (the container exposes on port 9000, available on the host as port 9999)</div>
               <br />
 
               <div># Check that the Docker container is running</div>
-              <CodeLine line={`docker ps`} />
+              <CodeLine index={4} line='docker ps' />
```

Comment:
See previous comments about how `index` should be zero.  Doesn't massively need to be an attribute.

Reply:
I think linting was complaining about it, but  can make it 0 for now if that still works